app/javascript/controllers/mapbox_controller.js

```diff
@@ -8,43 +8,67 @@ export default class extends Controller {
     this.map = new mapboxgl.Map({
       container: 'map-frame',
       style: 'mapbox://styles/mapbox/streets-v11',
-      center: [9.610, 59.178],
-      zoom: 9
+      center: [10.925860, 59.223840],
+      zoom: 11
```

Comment:
We should set a new center and zoom logic in a new PR. For now, it centers on the three schools added with the seed that runs on db:seed.